src/tests/createHeadlessForm.test.js

```diff
@@ -15,6 +15,7 @@ import {
   schemaInputTypeRadioOptionsWithDetails,
   schemaInputTypeSelectSoloDeprecated,
   schemaInputTypeSelectSolo,
```

Comment:
From what I see, this is ready to go! The only missing part is to test this against Remote's internal E2E tests ([internal guide](https://www.notion.so/remotecom/json-schema-form-Dev-Flow-eace1eee93fb40f7aff6cede07732a28?pvs=4#010993b6369c44c7918a1d879091a5d3)). If you need help, feel free to ping me by DM!

Reply:
Ah, you need to rever the package version to fix the pipelines, so I can approve it again.

Reply:
Reverted the version and edited the title. Waiting for pipelines to be 👍